index.html

```diff
@@ -1945,194 +2664,2171 @@
+                          In-depth Stories You Don't See Elsewhere About Topics
+                          You Want to Read!
+                        </strong>
+                        <p class="subscribe-description mt-3">
+                          All Access + Weekly Delivery. Choose A Membership
+                          That's Perfect for You!
+                        </p>
+                        <button
+                          class="subscribe-button button-tag-magazine-page px-2 rounded"
+                        >
+                          SUBSCRIBE
+                        </button>
+                      </div>
+                    </div>
+                  </div>
+                </div>
               </div>
-            </div>
-            <div class="hank hidden_mob">
-              <img src="./assets/hank-gilman.png" alt="hank"  />
-              <div>
-                <strong
-                  >Lessons From Enron 15 Years Later: The Greatest Fraud Trial
-                  Ever</strong
-                >
-                <div>BY <span class="gray-text">HANK GILMAN</span></div>
+              <div class="row">
+                <div class="col-md-12 pl-0 py-5 border-bottom">
+                  <p class="subscribe-title pl-2">NEWSLETTER</p>
+                  <div class="container pr-0">
+                    <div class="row">
+                      <div class="col-md-6 newsletter-end-image px-0 py-0">
+                        <img
+                          src="./assets/newsweek-bottom-img.JPG"
+                          alt="Newsweek image phone"
+                          class="img-fluid newsweek-bottom-img"
+                        />
+                      </div>
+                      <div class="col-md-6 border py-4 px-4">
+                        <strong class="newsletter-title">THE STARTING 5</strong>
+                        <p class="newsletter-description">
+                          See why nearly a quarter of a million subscribers
+                          begin their day with the Starting 5.
+                        </p>
+                        <div class="container">
+                          <div class="row">
+                            <div class="col-md-6 px-0">
+                              <p class="newsletter-email border px-3 py-0">
+                                Email address
+                              </p>
+                            </div>
+                            <div class="col-md-6 px-0">
+                              <button class="newsletter-button p-2">
+                                FREE SIGN UP
+                              </button>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
               </div>
             </div>
           </div>
         </div>
       </div>
     </main>
-
     <footer>

```

Comment:
- [ ] Let's improve the footer so it has the correct font weight, the separators between the top row and between each link and the social media icons correctly positioned.

Compare your footer with the original one below:
![Screenshot_2021-01-27 Bootstrap 4 Example(1)](https://user-images.githubusercontent.com/16193212/106026703-ea809680-60a8-11eb-8162-6dffc0ab2368.png)

![footer_desktop](https://user-images.githubusercontent.com/16193212/106026743-f5d3c200-60a8-11eb-9bad-213ea2daafde.png)

---- 

- [ ] Notice also how in the original site the background in the desktop version is a blue-ish grey color instead of white, make sure to implement the body background for your clone too.

----

![Screenshot_2021-01-27 Bootstrap 4 Example(3)](https://user-images.githubusercontent.com/16193212/106027597-cec9c000-60a9-11eb-90cf-228ac483db70.png)

- [ ] Both here, in the footer, and at the navbar you use a font that's similar to the Newsweek logo, but it's not the actual correct logo.

Logos and understanding how to handle them is very important, please look up and use the actual logo from Newsweek in your project, you can get the svg file from the original site or google for the logo png image with a transparent background.
Read more [here](https://deliciousbrains.com/svg-advantages-developers/) and [here](https://cheesecakelabs.com/blog/jpg-png-svg-web-beginners-guide/) about the types of images web developers should be familiar with.